tools/optimizer-server/src/main.rs

```diff
@@ -45,7 +45,7 @@ struct fanotify_event {
 struct EventInfo {
     path: String,
     size: u64,
-    timestamp: i64,
+    timestamp: u128,
```

Comment:
Keep the timestamp type in 'pkg/fanotify/conn/conn.go' the same as 'u128'?

Reply:
Also, can the 'timestamp' be changed to 'elapsed'?

Reply:
Unfortunately, golang does not have uint128 type. uint64 should be large enough

Reply:
renamed